models/src/main/java/com/vimeo/networking2/VideoInteractions.kt

```diff
@@ -0,0 +1,48 @@
+package com.vimeo.networking2
+
+/**
+ * All action that can be taken on a video.
+ */
+data class VideoInteractions(
+
+    /**
+     * The buy interaction for a On Demand video.
+     */
+    val buy: BuyInteraction? = null,
+
+    /**
+     * When a video is referenced by a channel URI, if the user is a moderator of the
+     * channel, include information about removing the video from the channel.
+     */
+    val channel: Interaction? = null,
+
+    /**
+     * Information about whether the authenticated user has liked this video.
+     */
+    val like: LikeInteraction? = null,
+
+    /**
+     * The Rent interaction for an On Demand video.
+     */
+    val rent: RentInteraction? = null,
+
+    /**
+     * Information about where and how to report a video.
+     */
+    val report: Interaction? = null,
+
+    /**
+     * Subscription information for an On Demand video.
+     */
+    val subscribe: Subscription? = null,
```

Comment:
sorry I wasn't clear on the previous comment that said "`subscription` would be more clear" but I was referring to the property being named `subscribe` being not clear since the class name was called `SubscriptionInteraction`. I think that `SubscriptionInteraction` was a find name for the class since every other interaction is named `XYZInteraction`.

Reply:
I update the field name and revert back to the class name.